File: src/client/java/com/somefrills/events/StartBreakingBlockEvent.java
package com.somefrills.events;

import net.minecraft.util.math.BlockPos;
import net.minecraft.util.math.Direction;

public class StartBreakingBlockEvent extends Cancellable {
    public BlockPos blockPos;
    public Direction direction;
    public StartBreakingBlockEvent(BlockPos blockPos, Direction direction) {
        this.blockPos = blockPos;
        this.direction = direction;
    }
}

File: src/main/kotlin/com/somefrills/events/StartBreakingBlockEvent.kt
package com.somefrills.events

import net.minecraft.core.BlockPos
import net.minecraft.core.Direction
import com.somefrills.events.FrillsEvent.Cancellable

class StartBreakingBlockEvent(val blockPos: BlockPos, val direction: Direction) : FrillsEvent(), Cancellable
